### PR TITLE
Even-odd training-eval selection. Trainings.yaml linked

### PR DESCRIPTION
**Pull request by ValVau**
Added trainings.yaml link into the code. Training of BDT on even events and evaluation on odd events added

### CODE REVIEW

File: cfg_VHcc_WLNu.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
                                                   f"{localdir}/params/triggers.yaml",
                                                   f"{localdir}/params/ctagging.yaml",
                                                   f"{localdir}/params/xgboost.yaml",
```
Comment (andreypz) [inline cfg_VHcc_WLNu.py]: Is xgboost file still needed? If not, then should be removed.

File: cfg_VHcc_ZLL.py [hunk 2]
```diff
@@ -26,6 +26,7 @@
                                                   f"{localdir}/params/triggers.yaml",
                                                   f"{localdir}/params/ctagging.yaml",
                                                   f"{localdir}/params/xgboost.yaml",
```
Comment (andreypz) [inline cfg_VHcc_ZLL.py]: Same here: xgboost file

**Review by andreypz — CHANGES_REQUESTED** [review]
This look fine to me.
Please address the comments, and resolve conflicts.

Also, I think you have the `model.txt` and `model_DY.txt` files in the root directory of the code which are not longer needed. If that's the case, could you remove those files from the repo.

File: cfg_VHcc_WLNu.py (hunk 1 above)
Comment (ValVau) [reply cfg_VHcc_WLNu.py]: Removed

File: cfg_VHcc_ZLL.py (hunk 2 above)
Comment (ValVau) [reply cfg_VHcc_ZLL.py]: Removed as well

**Comment by ValVau** [comment]
Added requested changes. Waiting for further comments/ merge

File: cfg_VHcc_ZLL.py (hunk 2 above)
Comment (andreypz) [reply cfg_VHcc_ZLL.py]: Looks like this is not done..

**Review by andreypz — CHANGES_REQUESTED** [review]
Please check again.

And what about my comment on the `model.txt` files?

**Comment by ValVau** [comment]
Please, have a look at latest changes

**Review by andreypz — APPROVED** [review]

**Pull request merged**